app/assets/javascripts/discourse/controllers/topic-bulk-actions.js.es6

```diff
@@ -26,6 +26,19 @@ export default Ember.Controller.extend(ModalFunctionality, {
   categoryId: Ember.computed.alias('model.category.id'),
 
   onShow() {
+    let showRelistButton = false;
+    const topics = this.get('model.topics');
+    for (let t = 0; t < topics.length; t++) {
```

Comment:
I think this could be replaced by something like 

`const showRelistButton = this.get('model.topics').some(t => !t.visible)`